x/concentrated-liquidity/math/tick.go

```diff
@@ -91,8 +82,9 @@ func TickToSqrtPrice(tickIndex, exponentAtPriceOne sdk.Int) (price sdk.Dec, err
 	return sqrtPrice, nil
 }
 
-// PriceToTick takes a price and returns the corresponding tick index
-func PriceToTick(price sdk.Dec, exponentAtPriceOne sdk.Int) (sdk.Int, error) {
+// PriceToTick takes a price and returns the corresponding tick index.
+// If tickSpacing is provided, the tick index will be rounded up to the nearest multiple of tickSpacing.
+func PriceToTick(price sdk.Dec, tickSpacing uint64) (sdk.Int, error) {
```

Comment:
Currently have an open question on if we need a roundUp boolean added to this. Otherwise it is always rounding to a higher tick

Reply:
Decided on today's call that we would just create an issue for this and proceed with this PR as is.

Reply:
Issue added here https://github.com/osmosis-labs/osmosis/issues/4979